emission/analysis/classification/inference/labels/inferrers.py

```diff
@@ -140,7 +143,9 @@ def n_to_confidence_coeff(n, max_confidence=None, first_confidence=None, confide
 
 # predict_two_stage_bin_cluster but with the above reduction in confidence
 def predict_cluster_confidence_discounting(trip, max_confidence=None, first_confidence=None, confidence_multiplier=None):
-    labels, n = lp.predict_labels_with_n(trip)
+    model_type = ModelType.GREEDY_SIMILARITY_BINNING
+    model_storage = ModelStorage.DATABASE
+    labels, n = eamur.predict_labels_with_n(trip, model_type, model_storage)
```

Comment:
the "n" is the number of rows of data that are associated with this prediction. that was the name and signature of the prediction function in the original code [here](https://github.com/e-mission/e-mission-server/blob/master/emission/analysis/modelling/tour_model_first_only/load_predict.py#L66). because you told me not to mess with anything outside of tour_model_first_only, i matched the signature and only changed which function was being called here.